api/v1beta1/activemqartemis_types.go

```diff
@@ -364,6 +364,8 @@ type DeploymentPlanType struct {
 	// Specifies the pod disruption budget
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Pod Disruption Budget"
 	PodDisruptionBudget *policyv1.PodDisruptionBudgetSpec `json:"podDisruptionBudget,omitempty"`
+
```

Comment:
it may need a comment like the attribute above it. just for consistency. And the csv annotation to give it a nice name.